Sources/CreateOptions/PackageDeclaration.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+
+public enum DependencyRule: Decodable {
```

Comment:
This is referred to as a `Requirement` by SPM, maybe let's match the terminology for consistency?

Reply:
Xcode uses the terminology "Dependency Rule" so I just matched it.

Reply:
Interesting 😄 it's called a `SourceControlRequirement` in the SPM source:

https://github.com/apple/swift-package-manager/blob/8970e016e9688f0e1fca549cdc02719fa5359059/Sources/PackageDescription/PackageRequirement.swift#L135-L191

For comparison, [XcodeGen uses 'Version'](https://github.com/yonaskolb/XcodeGen/blob/master/Docs/ProjectSpec.md#remote-package) and [Tuist appears to also use Requirement](https://github.com/tuist/tuist/blob/eede714adf9cf3910b6750f1efd439aff35515a8/Sources/ProjectDescription/Package.swift#L18-L35)

I think in this case, it's still better to match the SPM terminology to favour the actual source (and be a bit more considerate of Swift outside of Xcode) 🙇